packages/frontend/components/scaffold-eth/Balance.tsx

```diff
@@ -0,0 +1,81 @@
+import { BigNumberish, ethers } from "ethers";
+import { useEffect, useState } from "react";
+import { useBalance } from "wagmi";
+
+type BalanceProps = {
+  address: string;
+  price: number;
+  wrapperClasses?: string;
```

Comment:
Let's remove this for now. I think we could be useful in the future, but we don't have the need right now.